### PR TITLE
Add Partial app migration to migration doc for IIS folks

### LINKED ISSUE #10033: Add content for migrating app to ASP.NET Core piecemeal
A common pattern we've seen when migrating your app from ASP.NET to ASP.NET Core is to create an IIS sub-application and only move certain routes from ASP.NET to ASP.NET Core while preserving the URL structure of your application.

We should update this doc to mention this pattern and how to accomplish it.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1133d6d1-7801-654c-2b7c-dad2bb342501
* Version Independent ID: a7ccce22-f905-06f4-8847-b9eeaf534e1f
* Content: [Migrate from ASP.NET to ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/migration/proper-to-2x/?view=aspnetcore-2.2)
* Content Source: [aspnetcore/migration/proper-to-2x/index.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/migration/proper-to-2x/index.md)
* Product: **aspnet-core**
* GitHub Login: @isaac2004
* Microsoft Alias: **scaddie**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #10033

@shirhatti  can you edit this and complete the draft? Better to publish some help than to let the issue languish a few years.

### CODE REVIEW

File: aspnetcore/migration/proper-to-2x/index.md [hunk 1]
```diff
@@ -153,6 +153,40 @@ For example, an image asset in the *wwwroot/images* folder is accessible to the
...
+        <bindings>
+            <binding protocol="http" bindingInformation="*:80:" />
+            <binding protocol="https" bindingInformation="*:443:" sslFlags="0" />
+        </bindings>
+    </site>
+	...
+</sites>
+```
+
+And the directory structure:
```
Comment (guardrex) [inline aspnetcore/migration/proper-to-2x/index.md]: Optional nit:

```suggestion
Directory structure:
```

File: aspnetcore/migration/proper-to-2x/index.md [hunk 2]
```diff
@@ -153,6 +153,40 @@ For example, an image asset in the *wwwroot/images* folder is accessible to the
 
 [Multi-value cookies](xref:System.Web.HttpCookie.Values) aren't supported in ASP.NET Core. Create one cookie per value.
 
+## Partial app migration
+
+One approach is to create an IIS sub-application and only move certain routes from ASP.NET Framework to ASP.NET Core while preserving the URL structure the app. For example, consider the relevant section from the *applicationHost.config* file:
```
Comment (scottaddie) [inline aspnetcore/migration/proper-to-2x/index.md]:
```suggestion
One approach is to create an IIS sub-application and only move certain routes from ASP.NET 4.x to ASP.NET Core while preserving the URL structure the app. For example, consider the relevant section from the *applicationHost.config* file:
```

**Comment by Rick-Anderson** [comment]
@twsouthwick please review - unrelated to your email

File: aspnetcore/migration/proper-to-2x/index.md [hunk 3]
```diff
@@ -153,6 +153,40 @@ For example, an image asset in the *wwwroot/images* folder is accessible to the
 
 [Multi-value cookies](xref:System.Web.HttpCookie.Values) aren't supported in ASP.NET Core. Create one cookie per value.
 
+## Partial app migration
+
+One approach is to create an IIS sub-application and only move certain routes from ASP.NET 4.x to ASP.NET Core while preserving the URL structure the app. For example, consider the relevant section from the *applicationHost.config* file:
```
Comment (bariscaglar) [inline aspnetcore/migration/proper-to-2x/index.md]: URL structure _of_ the app

Comment (bariscaglar) [inline aspnetcore/migration/proper-to-2x/index.md]: This seems to beg for more explanation. "One approach" to what? I also expect the flow of the thought to continue after the relevant applicationHost.config section, since it started with "Consider".  Something like "The following section from applicationhost.config file is an example of how partial app migration can be achieved by moving the /api route but leaving out the others... From IIS perspective, I don't see any problem.

Comment (Rick-Anderson) [reply aspnetcore/migration/proper-to-2x/index.md]: @bariscaglar see my change. If that's not right, please suggest the entire line I can add.

Comment (Rick-Anderson) [reply aspnetcore/migration/proper-to-2x/index.md]: @bariscaglar any chance you could PR this and I'll close my PR?

Comment (bariscaglar) [reply aspnetcore/migration/proper-to-2x/index.md]: Sorry, I really can't. I am just here to give the IIS perspective since @shirhatti asked for it. After reading it from the point of view of a customer, I pointed out a couple of things that I found hard to follow in the documentation. I'll leave it to Sourabh to decide how to move on. From an IIS point of view I do not see any problem.
